rust-runtime/aws-smithy-http-server/src/plugin/mod.rs

```diff
@@ -119,6 +119,7 @@
 //!
 
 mod closure;
+pub mod either;
```

Comment:
I think I should probably pull `Either` into this module actually, rather than `pub mod`

Reply:
Yeah that could work except all these `Plugin`s are kinda `util`s though?

- `Stack`
- `FilterByOperationName`
- `OperationNameFn`